utils/hash.ts

```diff
@@ -1,7 +1,16 @@
 import * as crypto from "crypto";
+import { FamilyMembers } from "../generated/definitions/FamilyMembers";
 
 export const toHash = (s: string): string => {
   const hash = crypto.createHash("sha256");
   hash.update(s);
   return hash.digest("hex");
 };
+
+export const generateFamilyUID = (familyMembers: FamilyMembers): string =>
+  toHash(
+    Array.from(familyMembers)
+      .sort((a, b) => (a.fiscal_code > b.fiscal_code ? 1 : -1))
```

Comment:
what about using `a.localeCompare(b)` ?

Reply:
Ok for `localCompare`.
The regex allows only uppercase chars.

Reply:
Fixed in 3de19f0